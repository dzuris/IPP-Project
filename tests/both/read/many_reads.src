.ippcode22

DEFVAR GF@counter
MOVE GF@counter int@10

DEFVAR GF@num
LABEL start
JUMPIFEQ end GF@counter int@0

READ GF@num int
WRITE GF@num
WRITE string@\010
SUB GF@counter GF@counter int@1

JUMP start

LABEL end