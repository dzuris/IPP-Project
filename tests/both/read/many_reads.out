15
31
64
12
1789
164
1645
178
6485
1654
